src/v3/src/components/Widget/index.tsx

```diff
@@ -130,6 +131,15 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   const bootstrap = useCallback(async () => {
     await initLanguage();
     try {
+      if(typeof proxyIdxResponse !== 'undefined') {
+        setIdxTransaction({
+          // @ts-expect-error proxyIdxResponse has missing types
```

Comment:
nit: 
I would say its less of missing types and more incompatible types with `rawIdxState` and `context`

Reply:
is there a better way to pass this typecheck without `ts-expect-error` if it's not a missing types issue? Setting `idxTransaction` is a central part of the widget behavior so I am concerned about ignoring the types we pass to it.

Reply:
We could create a ticket for the SDK team to add these properties (which exist in the proxyIdxResponse) to the IdxTransaction interface.

Reply:
I think I'm a bit confused. The `ProxyIdxReponse` type is vastly different than `RawIdxState` or `IdxContext` types.

Reply:
Yep to both, or more like `context` and `RawIdxState` doesn't have the properties that `proxyIdxResponse` has. So if we just added those properties to `context` and `RawIdxState` in auth-js then we can avoid disabling the type check here.

Reply:
@leemchale-okta this is ok for now since we don't have a good way to solve this immediately before EA.

Please file a JIRA to fix these typings. If we keep it like this for too long we run the real risk of accidentally reading a property we think is safe to read from `IdxTransaction` but actually does not exist.

Reply:
Add the comment in both cases (here and below)

Reply:
okay will do 👍